src/bindings/Vscode.mli

```diff
@@ -378,7 +378,10 @@ module LanguageClient : sig
     }
   [@@bs.deriving abstract]
 
-  type processOptions = { env : string Js.Dict.t }
+  type processOptions =
+    { env : string Js.Dict.t
+    ; shell : bool
```

Comment:
Let's leave a comment here that this field in fact accepts the shell path as an argument as well.